verified-reviews/utils/client.ts

```diff
@@ -151,6 +151,7 @@ export const createClient = (params: ConfigVerifiedReviews | undefined) => {
             reviewRating: {
               "@type": "AggregateRating",
               ratingValue: Number(item.rate),
+              reviewCount: 1
```

Comment:
I could do it. But is really necessary? Because this will hapein just when is the review, so in that review could just exist one review

Reply:
At least for me, it appears to be a magic number. When it should be greater than 1? never? so your comment should be something as 
// reviewCount is always 1 because verified-reviews does not support multiple reviews .... (im guessing here, just an example)

Reply:
Right. I'll do it